website/docs/docs/get-started/privatelink/about-private-link.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "About PrivateLink"
+id: about-private-link
+description: "Configuring PrivateLink for AWS"
+sidebar_label: "About PrivateLink"
+---
+
+## What is PrivateLink?
+
+PrivateLink enables you to connect your data warehouse (DWH) hosted on AWS, in any region, to a dbt Cloud multi-tenant environment over the AWS backbone using the PrivateLink technology. PrivateLink allows dbt Cloud customers to meet security and compliance controls as it restricts information shared between dbt Cloud and the DWH from reaching the public internet. You can read more about AWS PrivateLink technology [here](https://aws.amazon.com/privatelink/).
+
+The initial release of support for AWS PrivateLink features integration with the following data warehouse providers:
+
+- Redshift
+- Snowflake
+- Databricks
+
+## Configuring PrivateLink
+
+Customers on all supported platforms will have to contact the dbt support team to complete the integration. It is not a self-service feature at this time. Ensure that the subject contains "Multi-tenant PrivateLink" in some form. Please allow 1-2 business days for our teams to complete the setup on our side. 
+
+### Setup
+
+dbt will provision the infrastructure pieces to ensure there are PrivateLink endpoints for a given account. Enabling this will ensure connections can be configured as “private” and will only communicate with the DWH over a PrivateLink. Once enabled, there will be two available DWH connection types in dbt Cloud: public and private. 
+
+Public connections are the classic connections that work today which communicate with the data warehouse over the public internet. Private connections are secured by PrivateLink and will always guarantee that all connections to the DWH are secured via AWS PrivateLink. Private endpoints are configured by dbt support on the backend once the infrastructure team provisions all the required services. Reach out to them with your organization name and which instance it's hosted in (EMEA, NA, AU).  
```

Comment:
```suggestion
Public connections are the classic connections that work today which communicate with the data warehouse over the public internet. Private connections are secured by PrivateLink and will always guarantee that all connections to the data platform are secured with AWS PrivateLink. dbt Labs support configures the private endpoints on the backend once the infrastructure team provisions all the required services. Please contact them with your organization name and which instance it's hosted in (such as EMEA, NA, and AU).  
```

Reply:
sugg:
- i don't like using "please" very much in product docs but this might a good use case here
- avoid latin words